### DISCUSSION TITLE
Where are the Strike Missions?

### DISCUSSION THREAD

**Discussion by GrEaSeLiTeNiN** [Khopa/dcs_liberation#1778] [Q&A] [answered]
Flying F18s and F16s in Caucasus campaign. I noticed the missions generated are always the usual CAS, BARCAP, DEAD, SEAD. I don't recall seeing any STRIKE missions generated yet. The Air Wing tab has STRIKE checked. But they don't seem to be generated.

**Comment by Starfire13** [comment, marked as answer]
The autoplanner does generate strike missions. However, you wouldn't want your strike flights flying into the threat zones of enemy SAMs, right? That's why the SEAD and DEAD flights are planned first. Also, perhaps your airfield is out of parking spaces or you are out of money.

**Reply by GrEaSeLiTeNiN** [reply]
Ok, thanks, will wait to see if the Strikes come on later.
